src/preset.js

```diff
@@ -0,0 +1,22 @@
+const defaultParseToMarkdown = require('./parse-rules/default/wikitext-to-markdown');
+const defaultParseToWiki = require('./parse-rules/default/markdown-to-wikitext');
+
+const htmlParseToMarkdown = require('./parse-rules/html/wikitext-to-markdown');
+const htmlParseToWiki = require('./parse-rules/html/markdown-to-wikitext');
+
+function getRulesForParseWikitextToMarkdown(preset) {
```

Comment:
need to have a white list of presets and turn this procedure into more strict form:
```
presets = [ 'presetA', 'presetB', 'default-with-html', ... ]
...
(preset) => {
  if (presets.indexOf(preset) === -1) {
    console.warn(`Invalid preset name ${preset}. Fallback to default`);
  }
  switch (preset) {
    case 'default-with-html':
      return [...defaultParseToMarkdown, ...htmlParseToMarkdown];
    default:
      return defaultParseToWiki;
  }
}
```